task_reactredux/task1/src/reducers/FormReducer.js

```diff
@@ -0,0 +1,15 @@
+import initialState from '../Constants';
```

Comment:
Actually, you could use not full state for each reducer. Just create constant object here, like 
```javascript
//FormReducer.js
const initialState = {
    wasSubmitted: false
}
//MailReducer.js
const initialState = {
    mail: '',
    mailValid: false,
};
//Same for password
```
CombineReducers will do the job